.travis.yml

```diff
@@ -0,0 +1,40 @@
+env:
+  global:
+    - RUST_BACKTRACE=1
+    - PATH=$PATH:$HOME/.cargo/bin
+os:
+  - linux
+  - osx
+language: rust
+rust:
+  - stable
+  - nightly
+matrix:
```

Comment:
We need to remove this now as travis has been updated to run nightly of 17 Nov (for it's nightly checks) which is compliant with clippy 0.0.99. So this should always pass henceforth like other crates. So remove the matrix to not allow failures on nightly any more